### DISCUSSION TITLE
What can I do to help?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#161] [Help] [answered]
I've been archiving thousands of url's through wayback machine's email service but that's about it. Can I merge PR's, something else?

**Comment by Hans5958** [comment]
I don't know, probably review pull requests?

**Comment by Explosion-Scratch** [comment]
I only know how to code python...

>

**Comment by Explosion-Scratch** [comment]
Otherwise I would be making PR's =P

>

**Comment by Hans5958** [comment]
No, I mean, reviewing pull requests, not creating them. 

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews

**Comment by NoobTracker** [comment]
I guess you can learn JS (it's pretty good) or ... well, look at things, think about things, stuff.

**Comment by Explosion-Scratch** [comment]
I already look and think about things lol

**Comment by NoobTracker** [comment]
well you can suggest things I guess

**Comment by Explosion-Scratch** [comment]
I also already do *that*.

**Comment by Hans5958** [comment]
I don't know, do some documentation, tell your friends, fix labels, triage issues, moral support, something?

**Comment by Explosion-Scratch** [comment]
lol idk, I've been using [this](https://www.iwebtool.com/link_extractor) and
emailing links to savepagenow@archive.org to archive everything on here and
on the first page of the scratch forums.

**Comment by Explosion-Scratch** [comment, marked as answer]
I guess I'll close this now.
